README.md

```diff
@@ -20,6 +32,9 @@ This repo is currently in beta testing. None of the code has been published yet,
 - [Directory structure](#directory-structure)
 - [Panel format](#panel-format)
 - [Median Pulse Height](#median-pulse-height)
+- [Development Notes](#development-notes)
+  - [Requirements](#requirements)
+- [Setup](#setup)
```

Comment:
This should probably be the same as requirements, a subheading under development notes right? If we want users to mainly use the conda environment probably best to make it clear Setup is only for developers.